packages/ra-core/src/util/FieldTitle.tsx

```diff
@@ -18,9 +18,15 @@ export const FieldTitle: FunctionComponent<Props> = ({
     isRequired,
 }) => {
     const translate = useTranslate();
+
+    if (label === false && label === '') {
```

Comment:
this doesn't make sense

Reply:
Ok. So the desired behavior is to return null only if label is false. Right? Correct me if I am wrong.

Reply:
I just followed below existing code - 
https://github.com/marmelab/react-admin/blob/master/packages/ra-ui-materialui/src/input/TextInput.tsx#L65
As per this code,  `FieldTitle` will not get rendered if string is empty.

Reply:
my bad, you're right. Both `''` and `false` should disable the title.